zerver/models.py

```diff
@@ -286,6 +286,9 @@ class Realm(models.Model):
     # Who in the organization is allowed to create streams.
     create_stream_policy: int = models.PositiveSmallIntegerField(default=POLICY_MEMBERS_ONLY)
 
+    # Who in the organization is allowed to delete their own message.
```

Comment:
"delete messages they themselves sent" would be clearer.

Reply:
Done